roofit/roofitcore/test/TestStatistics/testLikelihoodGradientJob.cpp

```diff
@@ -89,14 +105,14 @@ TEST_P(LikelihoodGradientJob, Gaussian1D)
 
    // --------
 
-   auto m0 = std::make_unique<RooMinimizer>(*nll);
+   RooMinimizer m0{*nll};
 
-   m0->setStrategy(0);
-   m0->setPrintLevel(-1);
+   m0.setStrategy(0);
+   m0.setPrintLevel(-1);
 
-   m0->migrad();
+   m0.minimize("Minuit2", "migrad");
```

Comment:
Can you explain this change? I guess it's not a big deal either way, but just curious if this makes a real difference.

Reply:
The `migrad()` function is there for backwards compatibility with the old RooMinuit class. For the new RooMinimizer that is more general, the intended entry point for the minimization is `minimizer()` with the user choices for the Minimizer algo